README.md

```diff
@@ -169,216 +190,17 @@ ArticleRepresenter.new(Article.new).to_json
-  end
-end
+### `virtual`
 
-example = {
-  'full_name' => 'Heidi Shepherd'
-}
+### `namespace`
 
-ExampleRepresenter.new(example).to_hash
+### `object`
 
-{
-  name: 'Heidi Shepherd'
-}
-
-```
-
-###3. Proc
-A Proc getter will be evaluated in the context of the object instance. Avoid using Proc getters for accessing
-methods or attributes on the object (this is done in the examples below for simplicity) and use the syntax above instead.
-
-```ruby
-class ExampleRepresenter
-  include Supa::Representable
-
-  define do
-    attribute :name, getter: proc { appellation.upcase }
-  end
-end
-
-class Person
-  def initialize(name)
-    @name = name
-  end
-
-  def appellation
-    "Mr or Mrs #{@name}"
-  end
-end
-
-example = Person.new('Heidi')
-
-ExampleRepresenter.new(example).to_hash
-
-{
-  name: 'MR OR MRS HEIDI'
-}
-
-```
+### `collection`
 
-Methods can also be defined on the Representer class to be referenced in a getter Proc:
-```ruby
-class ExampleRepresenter
-  include Supa::Representable
-
-  define do
-    attribute :name, getter: proc { appellation(0) }
-  end
-
-  def appellation(index)
-    "Dear #{@object.name.split(' ')[index]}"
-  end
-end
-
-class Person
-  def initialize(name)
-    @name = name
-  end
-
-  attr_reader :name
-end
-
-example = Person.new('Heidi Shepherd')
-
-ExampleRepresenter.new(example).to_hash
-
-{
-  name: 'Dear Heidi'
-}
-
-```
-
-###4. Literal Value
-Literal values can be supplied as `String`, `Numeric`, `Time`, `Date`, etc.
-```ruby
-class ExampleRepresenter
-  include Supa::Representable
-
-  define do
-    attribute :version, getter: 1.0
-    attribute :type, getter: 'documentation'
-    attribute :time, getter: Time.now
-  end
-end
-
-ExampleRepresenter.new({}).to_hash
-
-{
-  version: 1.0,
-  type: 'documentation',
-  time: 2017-01-09 14:45:00 +0100
-}
-```
-If a literal String clashes with a method name it must be wrapped in a proc (as per 3. above):
-```ruby
-  define do
-    attribute :type, getter: proc { 'articles' }
-```
-
-## `namespace`
-
-## `object`
-
-## `collection`
-
-### `:squash` option
-
-Passing `true` to `:squash` option results in merging collection with the previous one
-
-```ruby
-class AnimalsRepresenter
-  include Supa::Representable
-
-  define do
-    collection :animals, getter: -> { [{name: 'Rex', type: 'dogs'}] } do
-      attribute :name
-      attribute :type
-    end
-
-    collection :animals, getter: -> { [{name: 'Tom', type: 'cats'}] }, squash: true do
-      attribute :name
-      attribute :type
-    end
-  end
-end
-```
-
-```ruby
- AnimalsRepresenter.new(nil).to_hash
-```
-
-```ruby
-{
-  animals: [
-    {name: 'Rex', type: 'dogs'},
-    {name: 'Tom', type: 'cats'}
-  ]
-}
-```
-
-### `:getter` option
-
-Avoid passing Proc objects to `:getter` option because this is little slower than method name passing
-
-```ruby
-# Bad
-attribute :name, getter: -> { fetch_name }
-
-# Good
-attribute :name, getter: :fetch_name
-```
+### `append`
 
```

Comment:
Is the documentation going to be added or you would you like to do this as part of other story?